collections/ansible_collections/demo/patching/roles/build_report_network/tasks/main.yml

```diff
@@ -1,13 +1,22 @@
+- name: Create web directory if it does not exist
+  ansible.builtin.file:
+    path: "{{ file_path }}"
+    state: directory
+    mode: '0755'
+
 - name: create HTML report
   ansible.builtin.template:
     src: report.j2
-    dest: "{{ file_path }}/network.html"
+    #dest: "{{ file_path }}/network.html"
```

Comment:
yes, this should work when you include the landing page role

Reply:
Got it.